widgetssdk/src/test/java/com/glia/widgets/callvisualizer/controller/CallVisualizerControllerTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.glia.widgets.callvisualizer.controller
+
+import android.app.Activity
+import com.glia.widgets.callvisualizer.domain.IsCallOrChatScreenActiveUseCase
+import com.glia.widgets.core.dialog.DialogController
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.*
+
+internal class CallVisualizerControllerTest {
+    private lateinit var callVisualizerController: CallVisualizerController
+    private lateinit var isCallOrChatScreenActiveUseCase: IsCallOrChatScreenActiveUseCase
+    private lateinit var dialogController: DialogController
+
+    @Before
+    fun setUp() {
+        isCallOrChatScreenActiveUseCase = mock(IsCallOrChatScreenActiveUseCase::class.java)
+        dialogController = mock(DialogController::class.java)
```

Comment:
in kotlin, you can call `mock()`  without passing class to the method

Reply:
Hm, for some reason mock() doesn't work to me. 🤔

Reply:
Yeah! that's from the `mockito-kotlin` dependency added in branch `secure_conversations`, so it is impossible to use here and you can ignore my other comments related to `mockito` as well.